src/index.tsx

```diff
@@ -4,13 +4,43 @@ import { BrowserRouter } from 'react-router-dom'
 import './index.css';
 import App from './Components/App/App';
 import reportWebVitals from './reportWebVitals';
+import {
+  ApolloClient,
+  InMemoryCache,
+  ApolloProvider,
+  HttpLink,
+  from
+} from '@apollo/client';
 
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
+
+//pulled from Pedro example, need to figure out how to make TypeScript happy 
+
```

Comment:
Look into additional error handling via graphql - connect with BE about responses we could leverage.